backend/jsonlogic/filter.go

```diff
@@ -16,6 +19,39 @@ func opToDbOp(operator string) string {
 	}
 }
 
+// marshall escapes all occurrences of ' in string.
+func marshall(source string) string {
+	marshalled, _ := json.Marshal(source)
+	result := string(marshalled)
+	result = strings.Replace(result, "'", "\\'", -1)
+	result = strings.Trim(result, "\"")
+	return result
+}
+
+func getValueFromSource(source any) (string, error) {
+	switch v := source.(type) {
+	case map[string]any:
+		if prop, ok := v["var"]; ok {
+			switch vt := prop.(type) {
+			case string:
+				// enough to avoid sql injection? Columns shouldn't have any spaces anyway heh
```

Comment:
Maybe we can use https://pkg.go.dev/github.com/lib/pq#QuoteIdentifier and https://pkg.go.dev/github.com/lib/pq#QuoteLiteral ?

Reply:
Oh that's probably better yes